pyproject.toml

```diff
@@ -449,6 +449,7 @@ module = [
     "fastapi_utils.*",
     "sqlalchemy_utils.*",
     "sky.*",
+    "safetensors.*",
```

Comment:
I also see that you have removed `safetensors` as a dependency from this file. In this case, you need to add it to each integration that you have changed.

Reply:
Will do!